### PR TITLE
add delphix/openlab and add docker to download array

### PR DESCRIPTION
**Pull request by gdams**
##### Checklist

- [x] `npm test` passes
- [ ] documentation is changed or added
- [x] contribution guidelines followed [here](https://github.com/AdoptOpenJDK/openjdk-website/blob/master/CONTRIBUTING.md)

closes: https://github.com/AdoptOpenJDK/openjdk-website/issues/392

### CODE REVIEW

File: src/handlebars/sponsors.handlebars [hunk 1]
```diff
@@ -96,6 +97,7 @@
         <td><a target="_blank" href="https://inspiratechnologygroup.com/"><img src="dist/assets/inspira.png" alt="Inspira logo"></a></td>
         <td><a target="_blank" href="https://installbuilder.bitrock.com/"><img src="dist/assets/installBuilder.png" alt="installBuilder logo"></a></td>
         <td><a target="_blank" href="https://localizejs.com/"><img src="dist/assets/localize.png" alt="Localize logo"></a></td>
+        <td><a target="_blank" href="http://docs.openlabtesting.org/"><img src="dist/assets/openlab.png" alt="OpenLab logo"></a></td>
```
Comment (karianna) [inline src/handlebars/sponsors.handlebars]: Not https?

File: src/handlebars/releases.handlebars [hunk 2]
```diff
@@ -39,22 +39,29 @@
...
+          {{! Create each platform's selector tile }}
             \{{#each releases}}
               <div class="latest-asset latest-grid-border" id='latest-selector-\{{platform_name}}' onclick='selectLatestPlatform("\{{platform_name}}")'>
                 <img alt='\{{platform_official_name}} logo' src='\{{platform_logo}}' />
                 <h2 class="officialName"><var platform-name>\{{platform_official_name}}</var></h2>
                 <p class="releaseName"><var release-name>\{{release_name}}</var></p>
               </div>
             \{{/each}}
+            <a id="docker_link" href="" target="_blank">
+            <div class="latest-asset latest-grid-border" id='latest-selector-Docker'>
```
Comment (karianna) [inline src/handlebars/releases.handlebars]: What happens if there's no valid docker link for this release?

Comment (karianna) [reply src/handlebars/releases.handlebars]: I'm pretty sure there's an open issue saying that we're missing some right now.

Comment (gdams) [reply src/handlebars/releases.handlebars]: this might help us spot those issues then

**Pull request merged**